src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import mate.academy.util.HashUtil;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public User() {
+        this.salt = HashUtil.getSalt();

```

Comment:
don't do this here, it's not to call util class here, it should be called service class